k8s/crd/v12/Chart.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: v2
+name: sdp-operator-crd-v12
+description: A Helm chart for SDP Operator CRD v12
+
+# A chart can be either an 'application' or a 'library' chart.
+#
+# Application charts are a collection of templates that can be packaged into versioned archives
+# to be deployed.
+#
+# Library charts provide useful utilities or functions for the chart developer. They're included as
+# a dependency of application charts to inject those utilities and functions into the rendering
+# pipeline. Library charts do not define any templates and therefore cannot be deployed.
+type: library
+
+# This is the chart version. This version number should be incremented each time you make changes
+# to the chart and its templates, including the app version.
+# Versions are expected to follow Semantic Versioning (https://semver.org/)
+version: 0.1.0
```

Comment:
are we planning to have different versions here? Or when one is updated all the others get their version bumped as well?

Reply:
This version gets overridden to the operator `k8s/chart/Chart.yaml` chart version in github actions when we package and publish the chart. That's why this is untouched. But I can edit this too so it matches the operator chart version.